CONTRIBUTING.md

```diff
@@ -28,3 +28,30 @@ If you want to get in touch with us first before contributing, you can use:
 [dev-quick-start]: https://github.com/strimzi/strimzi-kafka-operator/blob/master/DEV_QUICK_START.md
 [release-list]: https://github.com/strimzi/strimzi-kafka-operator/blob/master/RELEASE.md
 [doc-contrib-guide]: https://strimzi.io/contributing/guide/
+
+# How to become a maintainer
+
+The governance of the project is defined in the [GOVERNANCE.md](https://github.com/strimzi/strimzi-kafka-operator/blob/master/GOVERNANCE.md) file in the Strimzi Github repository, but in summary certain members of the community are the "maintainers" who decide the direction of the project. New maintainers can be elected by a ⅔ majority vote of the existing maintainers.
```

Comment:
Just for my info: `⅔ majority` - so if there will be 4 maintainers, the majority is 3, which is not ⅔, should we consider this?

Reply:
A ⅔ majority means that 3 or 4 maintainers need to agree on the decision. It doesn't mean that the number of votes needs to exactly equal the majority threshold. I don't think we need or want to change that threshold.